ee/localserver/request-query.go

```diff
@@ -0,0 +1,49 @@
+package localserver
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+func (ls *localServer) requestQueryHandler() http.Handler {
+	return http.HandlerFunc(ls.requestQueryHanlderFunc)
+}
+
+func (ls *localServer) requestQueryHanlderFunc(res http.ResponseWriter, r *http.Request) {
```

Comment:
```suggestion
	return http.HandlerFunc(ls.requestQueryHandlerFunc)
}

func (ls *localServer) requestQueryHandlerFunc(res http.ResponseWriter, r *http.Request) {
```

Reply:
Usually signature is `w` and `r`. Or sometimes `resp` and `req` or `response` and `request`. But please don't mix.

Reply:
fixed